cypress/support/component-index.html

```diff
@@ -5,8 +5,11 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width,initial-scale=1.0">
-  <script src="https://accounts.google.com/gsi/client" async defer id="gsiClient"></script>
   <title>Components App</title>
+  <!-- AppCues Requirements  -->
+  <script type="text/javascript">
+    window.AppcuesSettings = { enableURLDetection: true };
+  </script>
```

Comment:
I'm open to testing suggestions here. I was unable to figure out a way to unit test the global `Appcues` window object.

Reply:
I think at some point it is worth trusting the library. We could add tests that verify we're setting the right values, but as far as appcues itself, I think that's probably out of our purview to test? But happy to have that contested